app/src/androidTest/java/org/mozilla/fenix/AppRequestInterceptor.kt

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix
+
+import android.content.Context
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.request.RequestInterceptor
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ui.robots.appContext
+
+class AppRequestInterceptor(private val context: Context) : RequestInterceptor {
```

Comment:
This is the class that allows us to workaround the issue with fxawebchannel extension not working for stage. Otherwise we can not sync using stage server

Reply:
I would add a comment to this class as it's not obvious, something like:
```
/**
 * Overrides the application's request interceptor to deactivate the FxA web channel 
 * which is not supported on the staging servers.
 */
```